packages/api/tests/lib/FileUtils/test-getChecksum.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const test = require('ava');
+const { getChecksum } = require('../../../lib/FileUtils');
+
+test('getChecksum() returns the value of the checksum property', (t) => {
+  const file = { checksum: 'asdf' };
+
+  t.is(
+    getChecksum(file),
+    'asdf'
+  );
+});
+
+test('getFileName() returns the value of the checksumValue property', (t) => {
```

Comment:
I think this test just needs to be deleted

Reply:
All of these descriptions should have read "getChecksum() ..." instead of "getFileName() ..."

Reply:
This test is not a duplicate.  One is making sure that the function will return the `checksum` property if it is set, the next is making sure that it will return the `checksumValue` property if it is set.

Reply:
Ah I see that now, change the value 'asdf' => 'asdfValue'? I might have just missed it because of the description being wrong.